Command/MigrateCommand.php

```diff
@@ -120,57 +98,35 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Query used to find elasticsearch records to import',
                 '{"sort":{"_uid":{"order":"asc"}}}'
+            )
+            ->addOption(
+                'dont-finalize',
+                null,
+                InputOption::VALUE_NONE,
+                'Don\'t finalize document'
```

Comment:
```suggestion
            ->addOption(
                'finalize',
                null,
                InputOption::VALUE_NONE,
                'Finalize document'
```

To comply with the requested variable in the code of execute. And it's safer to not finalize by default (because you can still do that afterwards). You can't undo finalize

Reply:
It's aligned with the migrate command (which finalize every document until now). We never had to not finalized them.